Formula/openssl@3.rb

```diff
@@ -108,6 +110,9 @@ def openssldir
   def post_install
     rm_f openssldir/"cert.pem"
     openssldir.install_symlink Formula["ca-certificates"].pkgetc/"cert.pem"
+    system "#{bin}/openssl", "fipsinstall",
+           "-out", "#{openssldir}/fipsmodule.cnf",
+           "-module", "#{lib}/ossl-modules/fips.dylib"
```

Comment:
Does this need to be deferred to `post_install`? What does this do when there is an existing `fipsmodule.cnf` in `openssldir`?

Reply:
Also, it looks like this step doesn't work on Linux, presumably because the extension isn't `.dylib`.

Reply:
> Also, it looks like this step doesn't work on Linux, presumably because the extension isn't `.dylib`.

I noticed other formulas hardcoding `.dylib`. Is there a preferred way to handle this more portably?

Reply:
```suggestion
    system "#{bin}/openssl", "fipsinstall",
           "-out", "#{openssldir}/fipsmodule.cnf",
           "-module", lib/"ossl-modules/#{shared_library("fips")}"
```

Reply:
fixed the dylib thing

Reply:
There are still unanswered questions here.